x-pack/plugins/uptime/public/components/functional/empty_state/empty_state_loading.tsx

```diff
@@ -9,28 +9,31 @@ import {
   EuiFlexGroup,
   EuiFlexItem,
   EuiLoadingSpinner,
+  EuiPanel,
   EuiTitle,
 } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import React from 'react';
 
-export const EmptyStateLoading = (props: any) => (
-  <EuiEmptyPrompt
-    title={
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiTitle size="l">
-            <h3>
-              {i18n.translate('xpack.uptime.emptyState.loadingMessage', {
-                defaultMessage: 'Loading…',
-              })}
-            </h3>
-          </EuiTitle>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiLoadingSpinner size="xl" />
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    }
-  />
+export const EmptyStateLoading = () => (
+  <EuiPanel>
+    <EuiEmptyPrompt
+      title={
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiTitle size="l">
+              <h3>
+                {i18n.translate('xpack.uptime.emptyState.loadingMessage', {
+                  defaultMessage: 'Loading…',
+                })}
+              </h3>
+            </EuiTitle>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiLoadingSpinner size="xl" />
```

Comment:
One more nit... This loading state would actually be better without the panel, with the loader animation on top of the "Loading...", and it would be better placed in the body of the prompt like:

```jsx
<EuiEmptyPrompt
    body={
      <Fragment>
        <EuiLoadingSpinner size="xl" />
        <EuiSpacer />
        <EuiTitle size="l">
          <h2>
                {i18n.translate('xpack.uptime.emptyState.loadingMessage', {
                  defaultMessage: 'Loading…',
                })}
          </h2>
        </EuiTitle>
      </Fragment>
    }
/>
```

Reply:
Added in https://github.com/elastic/kibana/pull/31663/commits/a29d7f418ffe9cef2da99e82ef099ca27545e0cf.